modules/ROOT/pages/_partials/intro-con.adoc

```diff
@@ -34,7 +34,7 @@ The {webapp} presents links to the applications available in a {PRODUCT_SHORT} c
 Most applications are available in the {webapp}, after you first log in.  However:
 
 * If *Red Hat AMQ Online* is displayed as *Not ready* in the *Applications* menu, click *Start service* to provision it.
-* You automatically have access to a shared instance of Fuse Online from the *Applications* menu, and if you create a {walkthrough-name}, you also provision an instance of Fuse Online which is not shared with other cluster users.
+* You automatically have access to a shared instance of Fuse Online from the *Applications* menu, and if you start a {walkthrough-name}, you also provision an instance of Fuse Online which is not shared with other cluster users.
```

Comment:
a little misleading as it only applies to some walkthroughs, I wonder should we be super specific

If you start the *Integrating message-oriented middleware with a RESTful API using AMQ Online* Solution Pattern, an instance of Fuse Online is provisioned which is not shared...

Reply:
How about:
* You automatically have access to a shared instance of Fuse Online from the *Applications* menu. If you start the _Integrating message-oriented middleware with a RESTful API using AMQ Online_ Solution Pattern, an instance of Fuse Online is provisioned which is not shared with other cluster users.

Reply:
WT2 also provisions the personal fuse :(

You automatically have access to a shared instance of Fuse Online from the Applications menu. An instance of Fuse Online, which is not shared with other cluster users, is provisioned automatically by starting some Solution Patterns. 

(if we get questions, we could then add the names of the walkthroughs)

Reply:
Sorry to join the conversation late! I think that might be slightly vague. As a user, I would find 'some Solution Patterns' a bit unclear.

Reply:
alternative is to list the names for WT1 and WT2 (and keep them in sync perpetually)

Reply:
ok, how about:

> You automatically have access to a shared instance of Fuse Online from the Applications menu. If you start the _Integrating message-oriented middleware with a RESTful API using AMQ Online_ Solution Pattern or the _Managing and protecting APIs using API management_ Solution Pattern, an instance of Fuse Online is provisioned which is not shared with other cluster users.

Reply:
minimalist?

You could put the wt names as bullets in under the text?